common.blocks/editor/editor.js

```diff
@@ -1,32 +1,95 @@
 modules.define('editor', ['i-bem__dom', 'jquery'], function(provide, BEMDOM, $) {
 
 provide(BEMDOM.decl(this.name, {
+    onSetMod: {
+        'js': {
+            inited: function() {
+                this.bindTo('keydown', this.onKeyDown);
+                this.keyKodes = { 49: 1, 50: 2, 51: 3, 52: 4, 53: 5, 54: 6, 55: 7, 56: 8, 57: 9 };
+            }
+        }
+    },
+
+    _insertTag: function(tag) {
+        var selection = window.getSelection(),
+            range = (selection.getRangeAt && selection.rangeCount) && selection.getRangeAt(0);
+
+        range.deleteContents();
+
+        var el = document.createElement('div'),
+            frag = document.createDocumentFragment(),
+            node, lastNode;
+
+        el.innerHTML = tag;
+
+        while ((node = el.firstChild)) {
+            lastNode = frag.appendChild(node);
+        }
+
+        range.insertNode(frag);
+
+        if (lastNode) {
+            range = range.cloneRange();
+            range.setStartAfter(lastNode);
+            range.collapse(true);
+
+            selection.removeAllRanges();
+            selection.addRange(range);
+        }
+    },
+
+    onKeyDown: function(e) {
+        var keyCode = this.keyKodes[e.keyCode];
+
+        if (e.altKey && keyCode) {
```

Comment:
inverse condition and use early return instead to avoid extra indentation bellow